source/jquery.flot.js

```diff
@@ -234,6 +234,7 @@ Licensed under the MIT license.
                 drawSeries: [],
                 drawAxis: [],
                 draw: [],
+                clickHoverFindNearby: [],
```

Comment:
So, instead of the hook route, I think we can accomplish a similar extensability experience through a couple of other new configuration options.

First, we can add a new option to the series options, called something like 'alwaysUsePointsForFindNearby', which would default to false. You would then change the logic at line 2352 (of your updated file) to be: 
```
if (s.lines.show || s.points.show || s.alwaysUsePointsForFindNearby)
```

For providing users of the hover plugin to find more than one datapoint, without having the hook we should plumb the 'computeDistance' delegate through the hover plugin's options. That is, the options would change from;

```
    var options = {
        grid: {
            hoverable: false,
            clickable: false
        }
    };
```

to

```
    var options = {
        grid: {
            hoverable: false,
            clickable: false
        },
        hover: {
            computeDistance: null // fn -> computeDistance(dx, dy)
        }
    };
```

Then, the plugin would pass that option on to the call to findNearbyItems.

Reply:
So, I think I need to weigh in on this, The reason I originally pushed the request for this hook was to allow plugins to integrate more closely with the hover plugin. My company uses a plugin which adds new items onto the plot which we want to act like it's built in to the main hover plugin. 

I'm certainly not the most knowledgeable on this subject, but from what I understood in my original pass on these changes, as well as the core hacks we implemented to get something working in the short term we would either have to implement a separate hover functionality into our plugin, or we'd need some sort of hook to allow it to interact nicely as I suggested originally in #1602

Reply:
Yeah, we just discussed your use case and I'm backtracking on my suggestion. Despite it being a departure from the traditional notion of a hook in flot, it seems like it might be the best approach for custom extension particularly for things like your use case. I'm going to make a slight change to the name of the hook, and then proceed with committing the change.